### PR TITLE
test: Add Windows SAC 2004 e2e

### PR DESCRIPTION
**Pull request by jsturtevant**
**Reason for Change**:

Adds Windows SAC 2004 e2e test.  This should have been a part of #3516 but the file was missing.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jsturtevant** [comment]
e2e tests for 2004 are passing on e2e infrastructure

File: test/e2e/test_cluster_configs/windows/sac/sac-2004.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+	"env": {
+		"REGION_OPTIONS": "westus2"
```
Comment (marosset) [inline test/e2e/test_cluster_configs/windows/sac/sac-2004.json:3]: Is there a reason to restrict this to westus2?

Comment (jsturtevant) [reply test/e2e/test_cluster_configs/windows/sac/sac-2004.json:3]: Not that I know of. It came from the other tests: https://github.com/Azure/aks-engine/tree/master/test/e2e/test_cluster_configs/windows

I will remove that restriction

Comment (marosset) [reply test/e2e/test_cluster_configs/windows/sac/sac-2004.json:3]: A lot of those tests used resources only available in westus2 (VHD, manage image, shared image gallery). Marketplace images should be available in all regions.

**Pull request merged**